Mathlib/Data/Fintype/Perm.lean

```diff
@@ -140,16 +140,26 @@ theorem card_perms_of_finset : ∀ s : Finset α, #(permsOfFinset s) = (#s)! :=
 def fintypePerm [Fintype α] : Fintype (Perm α) :=
   ⟨permsOfFinset (@Finset.univ α _), by simp [mem_perms_of_finset_iff]⟩
 
-instance equivFintype [Fintype α] [Fintype β] : Fintype (α ≃ β) :=
+instance Equiv.fintype [Fintype α] [Fintype β] : Fintype (α ≃ β) :=
```

Comment:
I think the convention here is `Equiv.instFintype`.

Reply:
I see. Does this mean my `MulEquiv.fintype` should also be `MulEquiv.instFintype`?